app/src/main/java/com/work/restaurant/view/search/bookmarks/SearchBookmarksFragment.kt

```diff
@@ -7,42 +7,75 @@ import android.view.View
 import android.view.ViewGroup
 import android.widget.Toast
 import androidx.recyclerview.widget.LinearLayoutManager
+import com.work.restaurant.Injection
 import com.work.restaurant.R
-import com.work.restaurant.data.repository.fitness.FitnessItemRepositoryImpl
-import com.work.restaurant.data.source.remote.fitness.FitnessCenterRemoteDataSourceImpl
-import com.work.restaurant.network.RetrofitInstance
-import com.work.restaurant.network.model.FitnessCenterItemResponse
+import com.work.restaurant.data.model.BookmarkModel
 import com.work.restaurant.view.adapter.AdapterDataListener
 import com.work.restaurant.view.base.BaseFragment
 import com.work.restaurant.view.search.bookmarks.adapter.BookMarkAdapter
 import com.work.restaurant.view.search.bookmarks.presenter.SearchBookmarksContract
 import com.work.restaurant.view.search.bookmarks.presenter.SearchBookmarksPresenter
 import com.work.restaurant.view.search.lookfor.SearchLookForActivity
-import com.work.restaurant.view.search.main.SearchFragment
 import kotlinx.android.synthetic.main.search_bookmarks_fragment.*
 
 class SearchBookmarksFragment : BaseFragment(R.layout.search_bookmarks_fragment),
-    View.OnClickListener, AdapterDataListener,
+    View.OnClickListener, AdapterDataListener.GetBookmarkData,
     SearchBookmarksContract.View {
 
+    private lateinit var bookMarkAdapter: BookMarkAdapter
+    private lateinit var presenter: SearchBookmarksPresenter
 
-    override fun showBookmarksList(fitnessList: List<FitnessCenterItemResponse>) {
-        recyclerview_bookmark.run {
 
-            this.adapter = bookMarkAdapter
+    override fun showBookmarkDeleteResult(msg: String) {
+        when (msg) {
+            RESULT_SUCCESS -> {
+                this.activity?.runOnUiThread {
+                    Toast.makeText(this.context, "제거성공.", Toast.LENGTH_LONG).show()
+                }
 
-            fitnessList.forEach { fitnessCenterItem ->
-                if (fitnessCenterItem.fitnessCenterLikeCount >= 90) {
-                    bookMarkAdapter.addData(fitnessCenterItem)
+            }
+            RESULT_FAILURE -> {
+                this.activity?.runOnUiThread {
+                    Toast.makeText(this.context, "제거실패.", Toast.LENGTH_LONG).show()
                 }
             }
-            layoutManager = LinearLayoutManager(this.context)
         }
+
     }
 
+    override fun getBookmarkData(select: Int, data: BookmarkModel) {
+        when (select) {
 
-    private lateinit var bookMarkAdapter: BookMarkAdapter
-    private lateinit var presenter: SearchBookmarksPresenter
+            SELECT_URL -> {
+                val intent = Intent(activity?.application, SearchLookForActivity()::class.java)
+                intent.putExtra("data", data.bookmarkUrl)
```

Comment:
```suggestion
                intent.putExtra(EXTRA_URL, data.bookmarkUrl)
```
상수로 관리해주시고 data라는 이름으로 관리하였는데, 
bookmarkUrl이라는 변수를 넣고있으므로 네이밍이 적절하지 않아보입니다.

Reply:
17db4cb 수정하였습니다.